dynamicio/core.py

```diff
@@ -227,6 +227,16 @@ def _apply_schema(self, df: pd.DataFrame) -> pd.DataFrame:
             raise ColumnsDataTypeError()
         return df
 
+    @staticmethod
+    def _transform_class_names_to_dataset_names(string_to_transform: str) -> str:
+        """Called by the __innit__ function to fetch dataset names from class name.
```

Comment:
typos: 
* innit -> init
* dataset names -> dataset name

I would rename the method to be `_transform_class_name_to_dataset_name` since it only takes one name and not an iterable

Reply:
Done: https://github.com/VorTECHsa/dynamicio/pull/24#:~:text=refactor%3A%20Rename%20method-,d2f0deb,-test%3A%20Use%20test